docs/Examples/watch-only-wallet.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const {HDPrivateKey, Network, Mnemonic} = require('bcoin');
+const {WalletClient} = require('bclient');
+
+(async () => {
+
+  // use well known test mnemonic
+  const phrase = [
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'abandon',
+    'about',
+  ].join(' ');
+
+  const network = Network.get('regtest');
+
+  const mnemonic = Mnemonic.fromPhrase(phrase);
+
+  // m'
+  const priv = HDPrivateKey.fromMnemonic(mnemonic);
+
+  // m'/44'
+  const bip44Key = priv.derive(44, true);
+
+  // m'/44'/0'
+  const bitcoinKey = bip44Key.derive(0, true);
+
+  // m'/44'/0'/0'
+  const accountKey = bitcoinKey.derive(0, true);
+
+  // account extended public key
+  // https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#Serialization_format
+  const xpub = accountKey.xpubkey(network.type);
+
+  // recommended to use hardware wallet to derive keys
+  // see github.com/bcoin-org/bledger
+
+  const client = new WalletClient({
+    network: network.type,
+    port: network.walletPort,
+  });
+
+  // create watch only wallet
+  // the wallet will generate lookahead
+  // addresses from the account extended public key
+  // and can find spendable coins in the blockchain state
+  const wallet = await client.createWallet('mywallet', {
```

Comment:
One thing I found helpful was to use `client.request(method, endpoint, params)` for these, so it's clear what endpoints are being used to follow along with https://bcoin.io/api-docs/index.html#create-a-wallet and the code.

Reply:
Maybe have comment above that has API call, so you can see both at once ?

Reply:
Yeah thats a good compromise -- I'll do that, there'll be both